mat64/list_test.go

```diff
@@ -161,7 +161,7 @@ func makeRandOf(a Matrix, m, n int) Matrix {
 			return &Vector{
 				n: 0,
 				mat: blas64.Vector{
-					Inc: 1,
+					Inc: 0,
```

Comment:
Just return &Vector{}; everything is zero, so there's no reason to state the zeros.